controllers/webhook/runcompletion_event_trigger_unit_test.go

```diff
@@ -117,6 +118,22 @@ var _ = Context("EventDataToPbRunCompletion", func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 
+		It("returns empty slices when there are no artifacts", func() {
+			rceWithoutArtifacts := rce
+			rceWithoutArtifacts.Artifacts = []common.Artifact{}
+			rceWithoutArtifacts.ServingModelArtifacts = []common.Artifact{}
+
+			protoRce, err := RunCompletionEventToProto(rceWithoutArtifacts)
+			expectedArtifacts := []*pb.Artifact{}
+
+			Expect(protoRce.Artifacts).To(Not(BeNil()))
+			Expect(protoRce.Artifacts).To(Equal(expectedArtifacts))
+			Expect(protoRce.ServingModelArtifacts).To(Not(BeNil()))
+			Expect(protoRce.ServingModelArtifacts).To(Equal(expectedArtifacts))
+
+			Expect(err).NotTo(HaveOccurred())
```

Comment:
Normally put this first straight after the call

Reply:
done in ed748638ca3535a869efeb8e7ab2088598fb64ad